src/NServiceBus.AcceptanceTests/Reliability/Outbox/When_sending_from_a_non_dtc_endpoint.cs

```diff
@@ -6,19 +6,20 @@
     using Configuration.AdvanceExtensibility;
     using EndpointTemplates;
     using NUnit.Framework;
-    using ScenarioDescriptors;
 
     public class When_sending_from_a_non_dtc_endpoint : NServiceBusAcceptanceTest
     {
         [Test]
-        public Task Should_store_them_and_dispatch_them_from_the_outbox()
+        public async Task Should_store_them_and_dispatch_them_from_the_outbox()
         {
-            return Scenario.Define<Context>()
+            Requires.OutboxPersistence();
```

Comment:
Just thinking out load. Technically it would be possible to pass something into the method to make that decision for us right? I do not think we need it just seeing if there is a way to address it.